### ISSUE TITLE
Error on load file with CURL

### ISSUE THREAD

**Issue by master-7** [jalik/jalik-ufs#40]
```
TypeError: Cannot call method 'match' of undefined
    at packages/jalik_ufs/ufs-server.js:125:1
    at b (domain.js:183:18)
    at Domain.run (domain.js:123:23)
    at Object.Package [as handle] (packages/jalik_ufs/ufs-server.js:91:1)
    at next (/opt/SaleGameTest/app/programs/server/npm/node_modules/meteor/webapp/node_modules/connect/lib/proto.js:174:15)

**Comment by jalik** [comment]
Thank you @master-7, the issue has been fixed in the v0.5.5.
Curl, wget and other tools do not send the "Content-Encoding" header, which was tested but undefined.

**Comment by master-7** [comment]
Thank you @jalik !
